base_ubl/models/ubl.py

```diff
@@ -124,21 +186,11 @@ def _ubl_add_party(
             self._ubl_add_language(partner.lang, party, ns, version=version)
         self._ubl_add_address(
             commercial_partner, 'PostalAddress', party, ns, version=version)
-        if commercial_partner.vat:
-            party_tax_scheme = etree.SubElement(
-                party, ns['cac'] + 'PartyTaxScheme')
-            registration_name = etree.SubElement(
-                party_tax_scheme, ns['cbc'] + 'RegistrationName')
-            registration_name.text = commercial_partner.name
-            company_id = etree.SubElement(
-                party_tax_scheme, ns['cbc'] + 'CompanyID')
-            company_id.text = commercial_partner.sanitized_vat
-            tax_scheme = etree.SubElement(
-                party_tax_scheme, ns['cac'] + 'TaxScheme')
-            tax_scheme_id = etree.SubElement(
-                tax_scheme, ns['cbc'] + 'ID', schemeID='UN/ECE 5153',
-                schemeAgencyID='6')
-            tax_scheme_id.text = 'VAT'
+        self._ubl_add_party_tax_scheme(
+            commercial_partner, party, ns, version=version)
+        if company:
```

Comment:
@alexis-via Do you remember if there was any particular reason why the PartyLegalEntity is only added for the partner who has the `res.company` in Odoo, eg. the sender of a sales invoice? In Netherlands the PartyLegalEntity is also required for the customer side.

Reply:
I don't think there is a good reason. I think it was linked to the fact that i decided to use the commercial_partner of the invoice as invoice address, which is probably a bad reason, cf https://github.com/OCA/edi/issues/104 As I said in this issue, I'm OK to modify that.

Reply:
@alexis-via Thanks for pointing me to the issue! I saw the fix for it was even already merged to the main 10.0 branch, so I backported it to the version I needed it for.